cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -22,6 +22,7 @@ def self.config(env)
     whitelisted_cookies = [
       'hour_of_code',
       'language_',
+      'partner',
```

Comment:
Oh I see, @davidsbailey already made the version change. I guess to confirm, that once his PR is merged in then my change will also take effect?